fairing/deployers/tfjob/tfjob.py

```diff
@@ -67,6 +67,6 @@ def get_logs(self):
         labels = {
             'tf-replica-index': '0',
             'tf-replica-type': 'worker',
-            'tf_job_name=': name
+            'tf-job-name': name
```

Comment:
due to https://github.com/kubeflow/tf-operator/commit/6f748507e0cafe787cb358b802176a53b6b11040

Reply:
thanks for catching this